crates/aranya-client/tests/tests.rs

```diff
@@ -1088,3 +1091,186 @@ async fn test_afc_monotonic_seq() -> Result<()> {
 
     Ok(())
 }
+
+/// This tests a bug where if the daemon is killed after registering net identifiers, they're not
+/// reloaded upon reboot, which causes create_bidi_channel to fail with "unable to lookup peer".
+#[cfg(feature = "afc")]
+#[test(tokio::test(flavor = "multi_thread"))]
+async fn test_afc_persist_net_identifier() -> Result<()> {
+    let interval = Duration::from_millis(100);
+    let sync_config = SyncPeerConfig::builder().interval(interval).build()?;
+    let sleep_interval = interval * 6;
+
+    let tmp = tempdir()?;
+    let work_dir = tmp.path().to_path_buf();
```

Comment:
nit: putting the boilerplate test code first makes tests a little more readable. 
```suggestion
    let tmp = tempdir()?;
    let work_dir = tmp.path().to_path_buf();

    let interval = Duration::from_millis(100);
    let sync_config = SyncPeerConfig::builder().interval(interval).build()?;
    let sleep_interval = interval * 6;
```
We aren't doing this in the other tests yet so feel free to ignore this. This could be cleaned up on a different PR.